docs/static/libs/algolia/algolia.min.css

```diff
@@ -1,467 +1,2 @@
-.searchbox {
```

Comment:
Im assuming that this diff makes this file identical to `master`? (Did you run `git co master static/libs/algolia/algolia.min.css`, then commit that?)

Reply:
yes, kinda! master currently has an edited version of mine of algolia.min.css -- so I git checkout-ed (haha) from the [last commit](https://github.com/expo/expo/commit/2a30c4fb4f65da6d9303f68936d30496c5e5c645#diff-c5ef6e0d71e5aa5afd0d72a862e27f3a) before mine